### ISSUE TITLE
Add support for customtemplate for programme-details

### ISSUE THREAD

**Issue by pmerkland** [MultinetInteractive/EduAdmin-WordPress#434]
Something like `[eduadmin-programme-detail customtemplate]`

I guess it would be good if we can keep the same format for all the parts as on the detail page for courses like `[eduadmin-detailinfo coursedescriptionshort]`, if not then we can do something like `[eduadmin-programmeinfo coursedescriptionshort]`

**Comment by pmerkland** [comment]
Might it even be an option to use the exact same detailview page och bookingview page for programme starts?

**Comment by itssimple** [comment]
> Might it even be an option to use the exact same detailview page och bookingview page for programme starts?

I'm not entirely sure what you mean by this. Do you mean using the normal course detail and booking view, or what are you proposing?

**Comment by pmerkland** [comment]
> > Might it even be an option to use the exact same detailview page och bookingview page for programme starts?
> 
> I'm not entirely sure what you mean by this. Do you mean using the normal course detail and booking view, or what are you proposing?

Yes, depending on how much work it would be. The data model is basically the same, some parameters have different names in the API though.

In the long run, we would probably save time as we don't have to maintain two almost identical copies of the pages.

OR we just develop support for custom template on the eduadmin-programme-detail page. Some of our customers already use a custom template for normal courses and want to do the same for programmes.

**Comment by itssimple** [comment]
That would be a totally different issue though, from custom templates on programmes.
So, for merging the booking page, I'd recommend splitting that in a separate issue.

No feature creeping.

**Comment by pmerkland** [comment]
I agree but if we merge the pages this issue for the custom template for programmes becomes irrelevant.
